kpi/utils/json.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django.core.serializers.json import DjangoJSONEncoder
+from django.utils.functional import Promise
+from django.utils.encoding import force_str
+from django.utils.text import normalize_newlines
+
+
+class LazyJSONEncoder(DjangoJSONEncoder):
+
+    def default(self, obj):
+        if isinstance(obj, Promise):
+            return force_str(obj)
+        return super().default(obj)
+
+
+class LazyJSONSerializable:
+    """
+    Wrapper for python objects (such as lists or dicts) which contain lazy gettext
+    objects and could not be interpreted when the application loads.
+
+    It can be JSON serialized.
+    """
+    def __init__(self, o):
+        self.object = o
+
+    def __repr__(self):
+        return f'<LazyJSONSerializable object={type(self.object)}>'
```

Comment:
just curious about the decision to hide the content here. i see the object returned by `gettext_lazy()` has this:
```
        def __repr__(self):
            return repr(self.__cast())
```
i don't know what `__cast()` does, but in effect you do get the value back out when calling `repr()`:
```
In [7]: thingy = gettext_lazy('stringy')

In [8]: type(thingy)
Out[8]: django.utils.functional.lazy.<locals>.__proxy__

In [9]: repr(thingy)
Out[9]: "'stringy'"
```

Reply:
During my tests, I got an `AppRegistryNotReady` error but I cannot reproduce it anymore. I'll return `__str__()` like it was at the very beginning.